client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -128,16 +113,11 @@ export class GoogleAnalyticsForm extends Component {
 		const showAnalyticsForStores =
 			config.isEnabled( 'jetpack/google-analytics-for-stores' ) &&
 			siteIsJetpack &&
-			wooCommerceActive &&
-			siteSupportsBasicEcommerceTracking;
+			wooCommerceActive;
 		const showAnonymizeIP =
-			config.isEnabled( 'jetpack/google-analytics-anonymize-ip' ) &&
-			siteIsJetpack &&
-			siteSupportsIPAnonymization;
+			config.isEnabled( 'jetpack/google-analytics-anonymize-ip' ) && siteIsJetpack;
 		const showEnhancedAnalyticsForStores =
-			config.isEnabled( 'jetpack/google-analytics-for-stores-enhanced' ) &&
-			siteIsJetpack &&
-			siteSupportsEnhancedEcommerceTracking;
+			config.isEnabled( 'jetpack/google-analytics-for-stores-enhanced' ) && siteIsJetpack;
```

Comment:
Can we remove the `jetpack/google-*` config flags, too? They are enabled in all environments except desktop. Is there a reason to disable them in the desktop Electron app? If I understand correctly, we're just editing and saving Jetpack settings here, not affecting analytics sent by Calypso, right? Cc @allendav who added this code in #17985 and #19888.

Reply:
Good call! I'll take care of that in this PR.

Reply:
The config flags have been completely removed in https://github.com/Automattic/wp-calypso/pull/30042/commits/1b9e675b3b91f082b4d05fbb04c6f48a86fb5a83.